recipes-app/switchserialmode/files/src/serial/cp2102n.c

```diff
@@ -238,12 +240,20 @@ static void cp2102n_hardware_reset(void)
 
 static void cp2102n_set_rs485_setup_time(uint8_t setuptime)
 {
-    cp2102n_cfg_operation->cfg_rs485_setup_time(cp2102n_cfg_operation, setuptime);
+    if (0 == GET_CP2102N_RS485_SETUP_TIME(cp2102n_cfg_operation->deviceConf)){
+        cp2102n_cfg_operation->cfg_rs485_setup_time(cp2102n_cfg_operation, DEFAULT_RS485_SETUP_TIME);
+    } else {
+        cp2102n_cfg_operation->cfg_rs485_setup_time(cp2102n_cfg_operation, setuptime);
+    }
```

Comment:
Set the default value in `cp2102_ops_init`, after `cp2102n_read_config`. Then here no change.

Also the code here does not have the time configured if only `-m 485` is provided without setup time and hold time parameter.

Reply:
The correct behavior should be once the mode is switched to RS485, these two time parameter must be configured to 0x05, or any other value that specified by the command line arguments.